keras_cv/losses/penalty_reduced_focal_loss.py

```diff
@@ -18,7 +18,7 @@
 # TODO(tanzhenyu): consider inherit from LossFunctionWrapper to
 # get the dimension squeeze.
 @tf.keras.utils.register_keras_serializable(package="keras_cv")
-class BinaryPenaltyReducedFocalXent(tf.keras.losses.Loss):
+class BinaryPenaltyReducedFocalCrossEntropy(tf.keras.losses.Loss):
```

Comment:
@tanzhenyu @lengzq Can either confirm that Xent is CrossEntropy?  If so this PR LGTM

Reply:
CrossEntropy